docs/getting-started.mdx

```diff
@@ -408,6 +437,68 @@ up. To run `integration_test/example_test.dart` on a connected Android, iOS or m
           internals depend on it.
       </Error>
 
+### FAQ
+
+<Accordion title="Testing fails with error inside 'integration_test/test_bundle.dart'">
+  The reason is probably a mismatch of `patrol` and `patrol_cli` versions. Make sure that you 
+  use the latest versions of both packages.
+</Accordion>
```

Comment:
Paste full error message – in a week we'll have no idea what the problem was.

Reply:
The thing is it's not a single type of error. It's always caused by the fact that test_bundler is generating `test_bundle.dart` which is not compatible with the `patrol` package but the errors thrown by the compiler are different

Reply:
Okay then 👍🏻